NAMESPACE

```diff
@@ -55,10 +56,17 @@ export(read_locality_file)
 export(read_simd_file)
 export(read_spd_file)
 export(start_fy)
+importFrom(dplyr,across)
 importFrom(dplyr,arrange)
+importFrom(dplyr,case_when)
+importFrom(dplyr,contains)
+importFrom(dplyr,group_by)
 importFrom(dplyr,if_else)
 importFrom(dplyr,mutate)
+importFrom(dplyr,rename)
 importFrom(dplyr,select)
+importFrom(dplyr,summarise)
+importFrom(dplyr,ungroup)
```

Comment:
Try and minimize the number of functions we're exporting, it's definitely best to use `package::function` rather fully importing the functions like this.

I'd say only import the function with `@importFrom` when you're using it *a lot* and/or it makes the code unreadable without it.